File: bto_management_system/model/hdb_manager.py
from .user import User
from common.enums import UserRole

class HDBManager(User):
    """Represents an HDB Manager user."""
    def __init__(self, name: str, nric: str, age: int, marital_status: str, password: str = "password"):
        super().__init__(name, nric, age, marital_status, password)

    def get_role(self) -> UserRole:
        return UserRole.HDB_MANAGER

File: BMS_J/J/Models/HDBManager.java
package Models;

import Enums.MaritalStatus;
import Enums.UserRole;

public class HDBManager extends User {

    public HDBManager(String nric, String password, String name, int age, MaritalStatus maritalStatus) {
        super(nric, password, name, age, maritalStatus);
    }

    @Override
    public UserRole getRole() { return UserRole.HDB_MANAGER; }

}